myStyle.css

```diff
@@ -0,0 +1,38 @@
+  }
+div p {
+    color: navy;
+    margin-left: 20px;
+  }
```

Comment:
the indentation is a bit awkward, I typically see something like this:
```
div p {
  color: navy;
  margin-left: 20px;
}
```
where the closing braces is lined up with the start of the opening line.  In this case, it is the `d` in `div`.